### PR TITLE
EI-83 - qualifying question page

### PR DESCRIPTION
**Pull request by gdbarnes**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            return View(new QualificationQuestionViewModel());
+        }
+
+        [Route("")]
+        [HttpPost]
+        public async Task<IActionResult> QualificationQuestion(string accountId, QualificationQuestionViewModel viewModel)
+        {
+            if (!viewModel.HasTakenOnNewApprentices.HasValue)
+            {
+                ModelState.AddModelError("HasTakenOnNewApprentices", "Select yes if you’ve taken on new apprentices that joined your payroll after 1 August 2020");
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs:24]: Can this error message go into a resource or constants file?

File: src/SFA.DAS.EmployerIncentives.Web.Tests/Controllers/ApplyController/QualificationQuestionTests/WhenNoAnswerIsSelected.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            var accountId = "ABC123";
+            var viewModel = new QualificationQuestionViewModel();
+
+            var result = await _sut.QualificationQuestion(accountId, viewModel);
+
+            var viewResult = result as ViewResult;
+            var returnedViewModel = viewResult.Model as QualificationQuestionViewModel;
+
+            returnedViewModel.Valid.Should().BeFalse();
+            returnedViewModel.Errors.Should().Contain(x => x.Key == "HasTakenOnNewApprentices" && x.Value == "Select yes if you’ve taken on new apprentices that joined your payroll after 1 August 2020");
```
Comment (David-B-Read) [inline src/SFA.DAS.EmployerIncentives.Web.Tests/Controllers/ApplyController/QualificationQuestionTests/WhenNoAnswerIsSelected.cs:32]: See comment on ApplyController about error message

File: src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs (hunk 1 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs:24]: What's the value of having them in a resource or constants file?

Comment (David-B-Read) [reply src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplyController.cs:24]: It's an approach used across most other ESFA repos I've encountered, you don't have magic strings in unit tests, only one place to update the content if/when wording changes, plus SonarCube won't moan at you ;)

**Pull request merged**